src/state/dispatchers/runner/RunnerDispatcher.js

```diff
@@ -15,3 +15,9 @@ export const dispatchStopRunner = (organizationId, workspaceId, datasetId) => ({
   workspaceId,
   datasetId,
 });
+
+export const dispatchUpdateRunner = (runnerId, newRunnerDetails) => ({
+  type: RUNNER_ACTIONS_KEY.TRIGGER_SAGA_UPDATE_RUNNER,
+  runnerId,
+  newRunnerDetails,
+});
```

Comment:
I can't see if we use the dispatcher somewhere ?

Reply:
In fact, there is no need a runner update. :rofl: